book/tuto-08-gouraud.md

```diff
@@ -42,10 +42,12 @@ If the light is *parallel* to the normal, then the surface should be bright. And
 ## In practice
 
 The main part of the calculation will be done in the fragment shader. However we need to modify
-the vertex shader first, in order to pass the normal's data to the fragment shader.
+the vertex shader first, in order to pass the normal's data to the fragment shader. In addition
+to this, also note that some hardware doesn't support some functions that we're going to use in
```

Comment:
How about:

> In addition to this, we need to specify a newer version of GLSL, since v140 doesn't support the functions we're going to use. To make the vertex shader work, we need at least GLSL v150.